eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -180,9 +180,12 @@ function UpdateDocsMsMetadataForPackage($packageInfoJsonLocation, $packageInfo)
   }
 
   # Copy package info file to the docs repo
+  $docsMsMetadata = &$GetDocsMsMetadataForPackageFn $packageInfo
+  $readMePath = $docsMsMetadata.LatestReadMeLocation
   $metadataMoniker = 'latest'
-  if ($originalVersion.IsPrerelease) {
+  if ($originalVersion -and $originalVersion.IsPrerelease) {
```

Comment:
This is technically unnecessary as powershell handles null objects for you without error, but at the same time adding the `-and` makes it clear you expect it could be null so I think it's ok.

```
$ if ($null.IsPrerelease) { echo success }
$
```

Reply:
We enabled the strict mode. That's why it complains for null case.

Reply:
Do we have valid cases where originalVersion is null?